woocommerce/woocommerce
# Fix "Are you sure you want to leave this page?" error

### Changes proposed in this Pull Request:





This fixes the problem of searching for countries and triggering the "Are you sure you want to leave this page?" error.



### How to test the changes in this Pull Request:

Pre-requisites for local testing:

1. Have woocommerce installed locally and synced within WooPayments. See Pe4Hk0-3f-p2 for details.
2. Enable the feature flag to see reactified settings page. pe4Hk0-1ly-p2.
3. Check out this branch.
4. Run pnpm i && pnpm run build

Then...
1. Go to Payments tab
2. Open the Countries dropdown
3. Search for something...
4. Close the dropdown
5. Try to refresh the page or click a provider
6. No error should be thrown



### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message

Fix the "Are you sure you want to leave this page?" bug when interacting with the countries' search input.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
